### DISCUSSION TITLE
1.1.26

### DISCUSSION THREAD

**Discussion by kirb** [wstyres/Zebra#1969] [Releases]
* Adds a handy feature to copy device info by tapping the device info text at the bottom of the home page.
* Fixes a crash on iOS 9 – 13 when tapping the Add Source button. (Contributed by @opa334 - thanks!)
* Fixes the sorting of the package list export feature to be alphabetical.

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.26'>1.1.26</a>.</em>

**Comment by lenick1996** [comment]
im having issues login into any repos for a while, anyone knows why that happens ?

**Reply by iAdam1n** [reply]
What happens exactly? Can you send a video?

**Reply by lenick1996** [reply]
https://user-images.githubusercontent.com/87044170/155918154-48af642e-2af3-4c20-9517-fcbdf6b39de7.MOV

**Reply by iAdam1n** [reply]
Thanks. That is weird. Can you login to either repo if you do it on Safari? I know that won't log you into them in Zebra but just to see if it works.

**Reply by lenick1996** [reply]
just wondering does it make a difference if im logged into chrome ? because all of them are logged on chrome

**Reply by iAdam1n** [reply]
Since the popup is Safari, you would need to try it in Safari.

**Reply by lenick1996** [reply]
i found the issue i had the block all cookies enabled deep in the safari settings sorry for wasting tour time

**Reply by iAdam1n** [reply]
No worries.
